new post about tickets

Marco-Santoni opened the pull request:
I wasn't able to make the post appear though
![image](https://github.com/pietroppeter/py4ai/assets/5029487/d3be5366-25fb-4d21-b706-da544f84250c)

Commit:
new post about tickets

pietroppeter commented:
I ran into the same problem and the explanation is that hugo does not publish a post in the future, it only shows it if it is in a datetime in the past (there is possibly a way to override it, I have not found it, nor actually even looked for it).

pietroppeter reviewed (changes requested):
The text looks good to me, as a minor point I would only make sure we are consistent with capitalization. As we mentioned yesterday we should also add the button with the new waiting list and remove the start date from the post.

Commit:
minor changes to new blog post

Commit:
add link to waitlist

Marco-Santoni commented:
- Fixed capitalization
- extended ticket to 14th of February
- added link to waitlist in blog post
- button on homepage is set automatically by Tito
